atm.js

```diff
@@ -1,21 +1,91 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
+//I want to deposit money into my checking account
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+    //gets the jquery deposit button element
+    var checkingDepositBtn = $("#checkingDepositBtn");
 
-}
+    //gets the jquery withdraw button element
+    var checkingWithdrawBtn = $("#checkingWithdrawBtn");
 
-function checkingWithdrawal(amount){
+    //getting the jquery checking balance element.
+    var checkingDisplay = $("#checkingAccount >.balance");
 
-}
+    //getting the current value of the balance
+    var checkingAmount = parseInt(checkingDisplay.html().replace("$", ""));
 
-function savingsDeposit(amount){
+    //Same for savings
+    var savingsDepositBtn = $("#savingsDepositBtn");
 
-}
+    var savingsWithdrawBtn = $("#savingsWithdrawBtn");
 
-function savingsWithdrawal(amount){
+    var savingsDisplay = $("#savingsAccount >.balance");
 
-}
+    var savingsAmount = parseInt(savingsDisplay.html().replace("$", ""));
```

Comment:
How could you use OOJS to create a constructor function that would save you the trouble of having to create variables that store buttons, inputs, etc. for each account.

If you need a place to start, trying passing in a DOM element as an argument to the constructor that contains the entire DOM representation of a given account.